File: IMS-UI/ClientApp/src/app/IMS.Models/Logging/ActivityLogsResponse.ts
import { Response } from "../Shared/Response";
import { ActivityLogs } from "./ActivityLogs";
import { PagingInfo } from "../Shared/PagingInfo";

export class ActivityLogsResponse extends Response{
    public activityLogRecords: ActivityLogs[];
    public pagingInfo : PagingInfo;
}

File: IMS-UI/IMS.Models/Logging/ActivityLogsResponse.cs
using IMS_UI.IMS.Models.Admin;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace IMS_UI.IMS.Models.Logging
{
    public class ActivityLogsResponse:Response
    {
        public List<ActivityLogs> ActivityLogRecords { get; set; }
        public PagingInfo PagingInfo { get; set; }
    }
}
